### ISSUE TITLE
After a legend is shown, it is sometimes not cleaned up

### ISSUE THREAD

**Issue by Albert634515** [beto-rodriguez/LiveCharts2#622]
**Describe the bug**
Legend is not cleaned up.
After a legend is shown, it is sometimes not cleaned up

**To Reproduce**
If the chart is displayed outside of the bounds of the graph and the mouse cursor goes over the legend and if the mouse cursor leaves the Legend and goes over the area where the graph is not, but the application is. Then the legend will not be removed.
If the mouse goes over the legend and then over the graph and then outside of the graph area, then the Legend is cleaned up.
In other words if I leave the graph with the mouse via the Legend, the Legend stays. If I leave via the graph area the legend is cleaned up. 

**Expected behavior**
Expected is that after I move the mouse outside the graph the legend also disapears

**Desktop (please complete the following information):**
 - Windows 11
 - LiveCharts 2.0 beta 400 (was already in previous version of livecharts 2.0)

**Comment by beto-rodriguez** [comment]
Hi and thanks!

I am afraid I do not fully understand, do you mean the tooltip?

**Comment by Albert634515** [comment]
No the whole legend stays

![image](https://user-images.githubusercontent.com/85439711/194105514-6bb9189d-911f-4d14-8bdc-34e8f9abc0c4.png)

**Comment by beto-rodriguez** [comment]
This is now fixed with https://github.com/beto-rodriguez/LiveCharts2/pull/768 and will be included in the next version fo the library, thanks for the report!
